js/app.js

```diff
@@ -1,48 +1,74 @@
-let output = document.getElementById("outputContainer")
-
+// Input : the first display line on the screen
 let input = document.getElementById("input")
 
+// Input : the second display line on the screen
+let output = document.getElementById("outputContainer")
+
 function display(num){
 
     if(input.value.length > 6){
-        document.getElementById("input").style.fontSize = "48px";
+        input.style.fontSize = "48px";
     }
     
     if(input.value.length > 12){
-        document.getElementById("input").style.fontSize = "28px";
+        input.style.fontSize = "28px";
     }
     
     if(input.value.length > 20){
         input.value = "";
     }
 
     input.value += num;
+ 
     console.log(input.value.length); 
     
 }
 
-function clear(){
+// Clear function called in AC button which clears the screen
+function Clear(){
     input.value = "";
-    document.getElementById("outputContainer").innerText = "";
+    output.innerText = "";
 }
 
+// Delete function called in <x| button which deletes the last enterd character on the screen
 function del(){
     input.value = input.value.slice(0, -1);
 }
 
+// Calculation Function 
 function calculate(){
     try{
+
+        /* 
+           eval function Converts a string to an integer evaluates it 
+           and provides the answer that is stored in outputValue which 
+           is of type number and if any error occurs in this eval it 
+           is caught in the catch block as Invalid
+        */
         let outputValue = eval(input.value);
-        
+
         if(outputValue>999999999999){
-            document.getElementById("outputContainer").style.fontSize = "30px";
+            output.style.fontSize = "30px";
+        }
+
+        if(outputValue === Infinity){
+            output.style.fontSize = "50px";
         }
 
-        document.getElementById("outputContainer").innerText = outputValue;
+        if(outputValue === Infinity){
+            output.style.fontSize = "50px";
+            output.innerText = "Error";
+        }
+        
+        if(outputValue !== NaN){
```

Comment:
Please add a Todo statement here since this comparison is not working as intended?